README.md

```diff
@@ -1,125 +1,224 @@
+- :angry: There are more users than resources, so they have to compete for it, but you don't know how to deal with that chaos
+- :ocean: Other popular tools are simply an overkill, have different purpose or require a lot of time to spend on reading documentation, installation and configuration (Graphana, Kubernetes, Slurm)
+- :penguin: People that are using your infrastructure expect only one interface for all the things related to training models (besides terminal): monitoring, reservation calendar and scheduling distributed jobs 
+- :collision: Can't risk messing up sensitive configuration by installing software on each individual machine, prefering centralized solution which can be managed from one place.
+
+2. You're a **standalone user** who has access to beefy GPUs scatterd across multiple machines.
+- :part_alternation_mark: You want to be able to determine if batch size is too small or if there's a bottleneck when moving data from memory to GPU - charts with metrics such as `gpu_util`, `mem_util`, `mem_used` are great for this purpose
+- :date: Visualizing names of training experiments using calendar helps you track how you're progressing on the project
+- :snake: Launching distributed trainings is essential for you, no matter what the framework is
+- :dizzy_face: Managing a list of training commands for all your distributed training experiments drives you nuts (because you didn't wrap them into bash scripts)
+- :zzz: Remembering to manually launch the training before going sleep is no fun anymore
+
+What TensorHive has to offer
+-----------------------------
+:zero: Dead-simple one-machine installation and configuration, no `sudo` requirements
+
+:one: Users can make GPU reservations for specific time range in advance via **reservation mechanism**
+
+&nbsp;&nbsp;&nbsp;&nbsp; :arrow_right: no more frustration caused by rules: **"first come, first served"** or **"the law of the jungle"**.
+
+:two: Users can prepare and schedule custom tasks (commands) to be run on selected GPUs and hosts
+
+&nbsp;&nbsp;&nbsp;&nbsp; :arrow_right: automate and simplify **distributed trainings** - **"one button to rule them all"**
 
-It's designed with __flexibility, lightness and configuration-friendliness__ in mind. 
+:three: Gather all useful GPU metrics, from all configured hosts **in one dashboard**
+
+&nbsp;&nbsp;&nbsp;&nbsp; :arrow_right: no more manually logging in to each individual machine in order to check if GPU is currently taken or not
+
+For more details, check out the [full list of features](#features)
 
 Getting started
 ---------------
 ### Prerequisites
-* Nodes should be accessible via SSH without password ([HOWTO set up SSH keys](https://www.ssh.com/ssh/keygen/))
-* Only NVIDIA GPUs are supported (```nvidia-smi``` is required)
+* All hosts must be accessible via SSH, without password, using SSH Key-Based Authentication ([How to set up SSH keys](https://www.shellhacks.com/ssh-login-without-password/)
```

Comment:
Is there a chance that some has machines which can be accessed by anyone (literally) also without any password - truly bad idea. I know that in some very simple, local scenarios this may be a thing, but maybe u're right...

Reply:
There can be another auth method used, like for example host-based.